src/componentLibrary/NewVaultsTable/NewVaultsTable.style.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+import { theme } from 'componentLibrary';
+
+const CoinPairWrapper = styled.div`
+  display: flex;
+  gap: ${theme.spacing.spacing2};
+  align-items: center;
+  font-weight: ${theme.fontWeight.medium};
+  white-space: nowrap;
+`;
+
+interface NumericValueProps {
+  highlight?: boolean;
+}
+
+const NumericValue = styled.div<NumericValueProps>`
+  color: ${(props) => (props.highlight ? theme.newVaultsTable.highlightColor : 'inherit')};
+  font-weight: ${theme.fontWeight.medium};
+`;
+
+const AddVaultButton = styled.div`
+  border-radius: 20px;
```

Comment:
Don't think we need this—Jay's bringing a lot of the element styles inline, so even if the CTA in the table design has a different radius, it's fine to go with the default.